Documentation/Setup/Config/Index.rst

```diff
@@ -2557,33 +2557,46 @@ sys\_language\_mode
          string
 
    Description
-         Setting various modes of handling localization.
-
+         Configures what system should do when a page is not translated
+         to the requested language.
+         It is only evaluated when `sys_language_uid` is greater than `0`,
+         and the requested page translation is NOT available.
+         Internally this setting influences the value set to the 
+         `$GLOBALS['TSFE']->sys_language_content`.
+         
          The syntax is "[keyword] ; [value]".
 
-         Possible keywords are:
+         **Possible keywords are:**
 
-         [default] - The system will look for a translation of the page (from
-         "Alternative Page Language" table) and if it is not found it will fall
-         back to the default language and display that.
+         [empty] - If not set, and the page is not translated, the system will
+         behave as if the default language was requested.
+         Internally both `$GLOBALS['TSFE']->sys_language_content`
+         and `$GLOBALS['TSFE']->sys_language_uid` whill be set to 0;
```

Comment:
only when the page translation is not available. E.g. you request L=3, there is no page overlay with for lang 3, so 
`$GLOBALS['TSFE']->sys_language_content` and `$GLOBALS['TSFE']->sys_language_uid` will both be set to 0.
Also see the note at the beginnig => "It is only evaluated when :ts:`sys_language_uid` is greater than `0` and the requested page translation is NOT available."